ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -407,9 +410,10 @@ private KsqlConfig(final boolean current,
     originalsWithPrefix(KSQ_FUNCTIONS_PROPERTY_PREFIX, false)
         .forEach((key, value) -> props.put(key, "[hidden]"));
 
-    configDef(true).names().forEach(
-        key -> props.put(key, ConfigDef.convertToString(values().get(key), typeOf(key)))
-    );
+    configDef(true).names().stream()
+        .filter(key -> !SSL_CONFIG_NAMES.contains(key))
```

Comment:
The idea here is to hide SSL configs when they are served over the HTTP API right? Consider doing ```filter(k -> !SSL_CONFIG_NAMES.contains(k))``` from KsqlResource in that case. We also use this function for other string-ifying of configs (e.g. logging) where it might be useful to have the ssl configs included.

Reply:
I've lot where else it is used and it makes sense in all occurrences, which seem to be putting listing properties and putting props into the command topic. Neither should have SSL config. So IMHO it's in the right place.

Reply:
Fair enough